packages/frontend/src/components/Explorer/ResultsCard/ColumnHeaderContextMenu.tsx

```diff
@@ -202,7 +198,7 @@ const ContextMenu: FC<ContextMenuProps> = ({
                     icon={<MantineIcon icon={IconTrash} />}
                     color="red"
                     onClick={() => {
-                        removeCustomDimension(getItemId(item));
+                        removeActiveField(getItemId(item));
```

Comment:
I understand the issue in the loom video also claims that is "easy" to delete a custom dimension because it might be referenced by a custom metric. 
Would adding a "warning" modal before deletion a good solution for this?

Reply:
I believe so 👍

Reply:
Created ticket and asked for feedback on potentially "cascading deletion" of the custom metric: https://github.com/lightdash/lightdash/issues/14589